doc/classes/DisplayServer.xml

```diff
@@ -62,7 +62,7 @@
 			<param index="1" name="tooltip" type="String" />
 			<param index="2" name="callback" type="Callable" />
 			<description>
-				Creates a new application status indicator with the specified icon, tooltip, and activation callback.
+				Creates a new application status indicator with the specified icon, tooltip, and activation callback. [param callback] will be called with two arguments: [int] mouse button index (one of [enum MouseButton] values) and [Vector2i] click position in screen coordinates.
```

Comment:
This may be more consistent with https://github.com/godotengine/godot/pull/69451 (not merged, do feel free to contest this, not sure about "take" either)
```suggestion
				Creates a new application status indicator with the specified icon, tooltip, and activation callback.
				[param callback] should take two arguments: the pressed mouse button (one of the [enum MouseButton] constants) and the click position in screen coordinates  (a [Vector2i]).
```